docs/guides/cloud/introduction.mdx

```diff
@@ -73,17 +75,18 @@ the latest results across the team, and identify areas of concern.
 
 ### Run tests in parallel, in priority order, or not at all
 
-With our [Smart Orchestration](/guides/cloud/smart-orchestration) features, you
-can run tests across a swarm of machines simultaneously while Cypress Cloud
-coordinates runners and
-[balances test loads](/guides/guides/parallelization#Balance-strategy) - no
-setup required! You can prioritize recently failed specs with
-[Spec Prioritization](/guides/cloud/smart-orchestration#Spec-Prioritization) to
+With our
+[Smart Orchestration](/guides/cloud/smart-orchestration/parallelization)
+features, you can run tests across a swarm of machines simultaneously while
+Cypress Cloud coordinates runners and
+[balances test loads](/guides/cloud/smart-orchestration/load-balancing#Balance-strategy) -
+no setup required! You can prioritize recently failed specs with
+[Spec Prioritization](/guides/cloud/smart-orchestration/spec-prioritization) to
 surface problems earlier, and cancel whole test runs on failure with
-[Auto Cancellation](/guides/cloud/smart-orchestration#Auto-Cancellation) to save
+[Auto Cancellation](/guides/cloud/smart-orchestration/run-cancellation) to save
```

Comment:
Are there plans to remove run cancelation content from the recorded runs page?
https://deploy-preview-5246--cypress-docs.netlify.app/guides/cloud/recorded-runs#Run-cancellation